tamr_client/_types/session.py

```diff
@@ -1,3 +1,42 @@
 import requests
 
-Session = requests.Session
+from tamr_client._types.auth import UsernamePasswordAuth
+
+
+class Session(requests.Session):
+    def __init__(self):
+        super(self.__class__, self).__init__()
+        self._stored_auth: UsernamePasswordAuth = UsernamePasswordAuth("", "")
```

Comment:
`UsernamePasswordAuth("", "")` is not an auth we ever expect to be valid. Could we change this type to be `Optional` and default to `None`? I think that more clearly captures that the session is not authorized by default.

Reply:
This was originally to satisfy `mypy`, but I believe code changes since then should make a `None` default possible.  Testing locally now.

Reply:
Done!